proprietary/Components/src/denhaag/unordered-list.tokens.json

```diff
@@ -0,0 +1,11 @@
+{
+  "denhaag": {
+    "unordered-list": {
+      "color": { "value": "{denhaag.color.grey.4}" },
+      "margin-inline-start": { "value": "0.5rem" },
+      "padding-inline-start": { "value": "0.5rem" },
+      "marker-font-size": { "value": "1rem" },
+      "lower-alpha-list-style-type": { "value": "lower-alpha" }
```

Comment:
There are five tokens defined here even though the css only uses three of them

Reply:
removed unused tokens in commit https://github.com/nl-design-system/denhaag/pull/767/commits/0c90ebf68b98f007db7a83e19f33b890b53e0e91